src/GregClient/NuGet/GregClient.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
```

Comment:
Once we start adding scripts to the NuGet package, we can no longer use the default `.nuspec` generated by NuGet. We have to manually construct the `.nuspec`. Here the version is set to 1.0.0, but this will be overridden by the powershell script which runs after build, setting the version number to the version number of Greg.dll.